### PR TITLE
Handle shape changes

### PR DESCRIPTION
**Pull request by orwonthe**
Adding optional title

### CODE REVIEW

File: hdav/hdav.py [hunk 1]
```diff
@@ -73,13 +76,15 @@ class HdavWindow(QWidget):
     MAX_LAYER_COUNT = len(OVERLAY_KEYS)
     OVERLAY_KEY_LABEL = "123456789ABCDEFGHIJKLMNOPQRSTUVWXYZ"
 
-    def __init__(self, layers, user_keys_callback, **kwargs):
+    def __init__(self, layers, user_keys_callback, legend_width, **kwargs):
         super().__init__(**kwargs)
+        if legend_width is None:
+            legend_width = 200
```
Comment (zach-waggoner) [inline hdav/hdav.py:82]: Could we just give `legend_width` a default value in the function header?

**Review by zach-waggoner — APPROVED** [review]
Looks good.

File: hdav/hdav.py (hunk 1 above)
Comment (orwonthe) [reply hdav/hdav.py:82]: It gets passed down through a lot of levels. I am not sure where the best place would be to establish the default value. So I left it at the bottom.

Comment (zach-waggoner) [reply hdav/hdav.py:82]: Right, but I'm not proposing that you change the level.

```
def __init__(self, legend_width=200):
    ...
```

is just a more succinct way of writing

```
def __init__(self, legend_width):
    if legend_width is None:
        legend_width = 200
```

Comment (zach-waggoner) [reply hdav/hdav.py:82]: Ah, never mind — I see that you're explicitly passing the `legend_width` as `None` above.

**Pull request merged**